src/KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro/KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro.csproj

```diff
@@ -1,22 +1,22 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-        <PackageId>KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro</PackageId>
-        <Description>Avro implementation for KafkaFlow serializer middleware using Confluent.SchemaRegistry.Serdes.Avro package</Description>
-        <RootNamespace>KafkaFlow.Serializer.SchemaRegistry</RootNamespace>
-    </PropertyGroup>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageId>KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro</PackageId>
+    <Description>Avro implementation for KafkaFlow serializer middleware using Confluent.SchemaRegistry.Serdes.Avro package</Description>
+    <RootNamespace>KafkaFlow.Serializer.SchemaRegistry</RootNamespace>
+  </PropertyGroup>
 
-    <ItemGroup>
-        <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />
-        <ProjectReference Include="..\KafkaFlow.SchemaRegistry\KafkaFlow.SchemaRegistry.csproj" />
-    </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\KafkaFlow.Abstractions\KafkaFlow.Abstractions.csproj" />
+    <ProjectReference Include="..\KafkaFlow.SchemaRegistry\KafkaFlow.SchemaRegistry.csproj" />
+  </ItemGroup>
 
-    <ItemGroup>
-        <PackageReference Include="Apache.Avro" Version="1.11.3" />
-        <PackageReference Include="Confluent.Kafka" Version="2.1.1" />
-        <PackageReference Include="Confluent.SchemaRegistry" Version="2.1.1" />
-        <PackageReference Include="Confluent.SchemaRegistry.Serdes.Avro" Version="1.9.3" />
-    </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Apache.Avro" Version="1.12.0" />
+    <PackageReference Include="Confluent.Kafka" Version="2.8.0" />
```

Comment:
Confluent.Kafka 2.9.0 was released 1 week ago. If there isn't any breaking change, I suggest we update it